libraries/cli/include/cli/default_config.hpp

```diff
@@ -42,6 +42,30 @@ const char *default_json = R"foo({
         "verbosity": "ERROR",
         "channels": [
           {
+            "name": "DAG_BLOCK_PH",
+            "verbosity": "INFO"
+          },{
+            "name": "DAG_SYNC_PH",
+            "verbosity": "INFO"
+          },{
+            "name": "GET_DAG_SYNC_PH",
+            "verbosity": "INFO"
+          },{
+            "name": "GET_PBFT_SYNC_PH",
+            "verbosity": "INFO"
+          },{
+            "name": "PBFT_BLOCK_PH",
+            "verbosity": "INFO"
+          },{
+            "name": "TRANSACTION_PH",
+            "verbosity": "INFO"
+          },{
+            "name": "DAGBLKMGR",
+            "verbosity": "INFO"
+          },{
+            "name": "DAGMGR",
+            "verbosity": "INFO"
+          },{
```

Comment:
Is there any reason why we should not change also testnet&devnet configs ?

Reply:
Actually this change made it in by mistake. I will revert it. I would like to first make these changes on devnet and test it on devnet before applying the changes to other networks as well